backend/compact-connect/lambdas/python/compact-configuration/handlers/compact_configuration.py

```diff
@@ -186,21 +187,25 @@ def _put_compact_configuration(event: dict, context: LambdaContext):  # noqa: AR
         validated_data['compactName'] = compact_name
 
         # Check if licenseeRegistrationEnabled is being changed from true to false
-        if validated_data.get('licenseeRegistrationEnabled') is False:
-            try:
-                existing_config = config.compact_configuration_client.get_compact_configuration(compact=compact)
-                if existing_config.licenseeRegistrationEnabled is True:
-                    logger.info(
-                        'attempt to disable licensee registration after it was enabled.',
-                        compact=compact,
-                        submitting_user_id=submitting_user_id,
-                    )
-                    raise CCInvalidRequestException(
-                        'Once licensee registration has been enabled, it cannot be disabled.'
-                    )
-            except CCNotFoundException:
-                # No existing configuration, so this is the first time setting this field
-                logger.info('No existing configuration, so this is the first time setting this field', compact=compact)
+        try:
+            existing_config = config.compact_configuration_client.get_compact_configuration(compact=compact)
+            if (
+                validated_data.get('licenseeRegistrationEnabled') is False
+                and existing_config.licenseeRegistrationEnabled is True
```

Comment:
Are `is False` and `is True` necessary/intentional, or can we just write something like...

```suggestion
                not validated_data.get('licenseeRegistrationEnabled')
                and existing_config.licenseeRegistrationEnabled
```

Reply:
On second look, perhaps the first `is False` is necessary since `.get` can return `None`?

Reply:
The field is required and cannot be null, but I still prefer to be explicit here in the case of booleans so there is no room for ambiguity, if that works for you.